### ISSUE TITLE
After download from web page it says virus detected

### ISSUE THREAD

**Issue by donnyv** [LukeEmmet/GemiNaut#20]
I went here to download the install file and Windows said there was a virus detected.
https://www.marmaladefoo.com/pages/geminaut#head6

**Comment by LukeEmmet** [comment]
Hello - thanks for the feedback. It is a false positive. Because GemiNaut is not widely downloaded, many antivirus products do not consider it to be high trust. I now sign the executable and installer, so if you are unsure, please check it is signed.

there was some previous discussion of this problem on a previous ticket: https://github.com/LukeEmmet/GemiNaut/issues/7

**Comment by donnyv** [comment]
Windows Defender won't even let me use it. As soon as its downloaded it deletes it and flags it.

**Comment by LukeEmmet** [comment]
yes it is frustrating :( It may be because GemiNaut uses TLS to access network resources among other things. I dont have Windows Defender enabled on my machine (I use an alternative AV) but you can add exclusion I think: https://support.microsoft.com/en-us/windows/add-an-exclusion-to-windows-security-811816c0-4dfd-af4a-47e4-c301afe13b26

**Comment by LukeEmmet** [comment]
Or if you wish, you can compile it yourself if you have any concerns. I hope this helps
